src/types/streaming_server/settings.rs

```diff
@@ -7,6 +7,7 @@ pub struct Settings {
     pub cache_root: String,
     pub server_version: String,
     pub remote_https: Option<String>,
```

Comment:
Would be nice to include some documentation for remote_https as well, e.g. what this field means and 1-2 examples

Reply:
usually - yes, we should. At least we need to know what it does and what is it usescase.
let's leave it without a documentation for this time.